understanding/contrast-enhanced.html

```diff
@@ -135,223 +130,113 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Text can be seen by people who need strong contrast.</dd>
+                  <dt>目標</dt>
+                  <dd>テキストを、強いコントラストが必要な人にも見られるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Strongly contrast text against its background.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>背景に対してテキストを強くコントラストさせる。</dd>
```

Comment:
「コントラストさせる」はかなり違和感があります。
Strongly contrastを「コントラストを強める」「コントラストを強くする」のようにすると成立するのではないでしょうか。たとえばこう?

```suggestion
                  <dd>背景に対するテキストのコントラストを強くする。</dd>
```